src/assets/i18n/en_US.json

```diff
@@ -884,9 +884,37 @@
   "product.ready_for_shipment.within24": "Usually ready for shipment within 24 hours",
   "product.remove_all_product_filters.text": "Reset all filters",
   "product.retailSet.products.heading": "This retail set contains",
+  "product.review.content.error.required": "Please enter a review content.",
+  "product.review.content.label": "Review",
+  "product.review.content.max_limit": "Your review: {{0}} characters remaining.",
+  "product.review.content.placeholder": "What did you like or dislike? Why did you buy the product?",
+  "product.review.name.label": "Name",
+  "product.review.rating.error.required": "Click stars to rate the product.",
+  "product.review.rating.label": "Rating",
+  "product.review.rating.tooltip": "{{0, plural, one{# star} other{# stars}}}",
+  "product.review.status.new.text": "Your review is currently being checked. Other users will not be able to see your review until it has been approved.",
```

Comment:
Please change to: "Other users will not be able to see your review until it will have been approved."

Reply:
Doesn't "until it has been approved" sound a little bit cleaner/less complicated?

Reply:
"...will have been approved" is grammatically right in terms on tenses. Using present tense for future is working fine in German but not necessarily in English.